File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1607.cpp
#include <iostream>

using namespace std;

int main(){
    int n;
    cin >> n;
    
    while(n--){
        string a, b;
        cin >> a >> b;
        
        int tot = 0;
        for(int i = 0; i < a.size(); i++){
            
            if(a[i] <= b[i]){
                tot += b[i] - a[i];
            } else {
                tot += (26 + b[i]) - a[i];
            }
            
        }

        cout << tot << endl;

    }
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1607.py
n = int(input())

for i in range(0, n):
    f = input().split()
    a = f[0]
    b = f[1]
    
    tot = 0
    for j in range(0, len(a)):
        if(a[j] <= b[j]): tot += ord(b[j]) - ord(a[j])
        else: tot += 26 + ord(b[j]) - ord(a[j]);
    
    print(tot)